Src/FluentAssertions.Core/Core.pcl.csproj

```diff
@@ -30,11 +30,11 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>..\..\Artifacts\Lib\portable-monotouch+monoandroid+xamarin.ios\</OutputPath>
+    <OutputPath>bin\Release\</OutputPath>
```

Comment:
So if I understand this correctly, this project is no longer a separate deliverable of the NuGet package because it got superseded by the netstandard package. And it is now only needed for the versions of FA targeting the older frameworks? Right?